### PR TITLE
Fallback to sponsors JSON if no sponsors in prismic

### PR DESCRIPTION
**Pull request by clari182**
- Preview with sponsor content coming from Prismic: https://deploy-preview-2211--staging-aiid.netlify.app/
- Preview with fallback content (local json): https://deploy-preview-25--clarayoudale-staging.netlify.app/

### CODE REVIEW

File: site/gatsby-site/src/components/landing/Sponsors.js [hunk 1]
```diff
@@ -128,26 +164,29 @@ export default function Sponsors({ sponsors = [] }) {
         </div>
       </div>
       {sponsors.map((sponsor) => {
-        return sponsor?.node?.data?.items.map((item) => {
+        return sponsor?.items.map((item) => {
+          const text = item.richText
+            ? item.richText
+            : t(`sponsor-${item.name}`).replace(
+                /\[\[((.*?))\]\]/g,
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/landing/Sponsors.js:171]: Woot, what does this do?

File: site/gatsby-site/src/components/landing/sponsors.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            }
+        ]
+    },
+    {
+        "name": "Database Founding Sponsor",
+        "items": [
+            {
+                "name": "Partnership on AI",
+                "modalName": "PAI",
+                "text": "As a [[founding sponsor]] of the AI Incident Database (AIID), Partnership on AI supported the start-up of the database, including a grant to support the project prior to the founding of the Responsible AI Collaborative. Partnership on AI also invested in promoting the social benefits of the AIID to their Partners and other stakeholders and have an ongoing interest in supporting its outcomes.",
```
Comment (cesarvarela) [inline site/gatsby-site/src/components/landing/sponsors.json:21]: It looks like you invented some syntax to be able to have links. Maybe use markdown and use the already existing `react-markdown` component?

File: site/gatsby-site/src/components/landing/Sponsors.js (hunk 1 above)
Comment (cesarvarela) [reply site/gatsby-site/src/components/landing/Sponsors.js:171]: I see what it does, see my comment below.

**Comment by clari182** [comment]
Hey @cesarvarela can you review this PR please? Thanks!

**Review by cesarvarela — APPROVED** [review]

**Comment by clari182** [comment]
@cesarvarela can we merge this?

**Pull request merged**